### ISSUE TITLE
No `getContinuation` for playlists

### ISSUE THREAD

**Issue by aparatext** [LuanRT/YouTube.js#70]
## Expected Behavior
Since playlists can often be extremely long, I expected `getContinuation` to work, but it seems to not yet be implemented.

### Steps to Reproduce
```
const wl = await youtube.getPlaylist('WL');
console.log(wl.getContinuation());
```

## Checklist
- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I have provided sufficient information

**Comment by LuanRT** [comment]
Yup, not implemented yet.
Will probably work on that later today or maybe tomorrow.

**Comment by walking-octopus** [comment]
Thanks for your hard work!

**Comment by Wykerd** [comment]
This is fixed with #44 which just merged!
